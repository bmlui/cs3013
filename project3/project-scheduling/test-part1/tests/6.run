./scheduler 0 SJF tests/6.in 0
