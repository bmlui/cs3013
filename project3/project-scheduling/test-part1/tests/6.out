Execution trace with SJF:
t=0: [Job 2] arrived at [0], ran for: [1]
t=1: [Job 13] arrived at [0], ran for: [2]
t=3: [Job 14] arrived at [0], ran for: [4]
t=7: [Job 4] arrived at [0], ran for: [5]
t=12: [Job 12] arrived at [0], ran for: [5]
t=17: [Job 1] arrived at [0], ran for: [6]
t=23: [Job 5] arrived at [0], ran for: [6]
t=29: [Job 6] arrived at [0], ran for: [8]
t=37: [Job 9] arrived at [0], ran for: [10]
t=47: [Job 0] arrived at [0], ran for: [11]
t=58: [Job 16] arrived at [0], ran for: [12]
t=70: [Job 7] arrived at [0], ran for: [13]
t=83: [Job 3] arrived at [0], ran for: [14]
t=97: [Job 19] arrived at [0], ran for: [14]
t=111: [Job 15] arrived at [0], ran for: [17]
t=128: [Job 17] arrived at [0], ran for: [19]
t=147: [Job 8] arrived at [0], ran for: [20]
t=167: [Job 10] arrived at [0], ran for: [20]
t=187: [Job 11] arrived at [0], ran for: [20]
t=207: [Job 18] arrived at [0], ran for: [20]
End of execution with SJF.
